duckduckgo/Android
# Add feedback and broken site form to android

Task/Issue URL: https://app.asana.com/0/361428290920652/688515420618259
Tech Design URL: N/A

**Description**:
Add a native feedback form for general and broken sites feedback.

Note: the UI uses native text fields, I will be validating that with design and expect to do a final pass to polish.

**Steps to test this PR**:
TEST 1: Broken site:
1. From the browser page, select the "Report Broken Site"
1. Note that the url is filled in and submit button active
1. Submit the feedback
1. Ensure it appears in https://app.asana.com/0/569473074191537/717268129349527
1. Check the url and device details in the Asana task are correct
1. Now delete it from Asana!

TEST 2: Broken site validation:
1. From the browser page, select the "Report Broken Site"
1. Note that the url is filled in and submit button active
1. Delete the url
1. Note that the submit button is now inactive
Note: we only do basic "empty" validation here (the same as extensions) as a user may just want to enter text e.g "facebook"

TEST 3:  Feedback Message:
1. From the settings page, select "Leave Feedback"
1. Note that the submit button is inactive
1. Enter message text 
1. Submit the feedback
1. Ensure it appears in https://app.asana.com/0/413877547933099/717275776651708
1. Check the message and device details in the Asana task are correct
1. Now delete it from Asana!

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
